src/database/query/users/getSingleMemberQuery.ts

```diff
@@ -0,0 +1,35 @@
+import connection from '../../config/connection';
+import { MemberIdQueryArgs } from '../../../utils';
+
+const getSingleMemberQuery = ({ memberId }: MemberIdQueryArgs) => {
+  const sql = {
+    text: `
+      SELECT
+        users.full_name,
+        users.username,
+        users.email,
+        users.github,
+        users.linkedin,
+        users.avatar,
+        TO_CHAR(users.date_of_birth, 'YYYY-MM-DD') AS date_of_birth,
+        career_status.name AS career_status,
```

Comment:
No need to add `As career_status`. Just add it if you want to edit the column name when returning the data